### PR TITLE
Release notes for Forms 8.13.14, 10.5.4, 12.2.3 and 13.0.2 and Deploy 4.9.4, 10.3.4, 12.1.4 and 13.0.4

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added release notes for Forms 8.13.14, 10.5.4, 12.2.3 and 13.0.2 and Deploy 4.9.4, 10.3.4, 12.1.4 and 13.0.4

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [X] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 8.13.14, 10.5.4, 12.2.3 and 13.0.2 and Deploy 4.9.4, 10.3.4, 12.1.4 and 13.0.4

## Deadline (if relevant)

By Thursday this week if possible please (doesn't matter if earlier).

### CODE REVIEW

**Comment by AndyButland** [comment]
Linter reports " We prefer 'primary' over 'master.'" - but "master" [is the term still used in Umbraco for templates](https://docs.umbraco.com/umbraco-cms/tutorials/creating-a-basic-website/creating-master-template-part-1).

**Comment by alina-tincas** [comment]
Hi @AndyButland thank you for the PR 🙌 

We will take a look at it as soon as we can 😊

File: 10/umbraco-forms/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,15 @@ This section contains the release notes for Umbraco Forms 8 and 10 including all
 
 <summary>Version 10</summary>
 
+[**10.5.4**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.5.4) **(February 20th 2024)**
+
+* Fixed ordering of forms by name in security screen [#1122](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1122)
+* Update the copy form dialog to use standard CMS patterns for button state and disabling after click [#1121](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1121).
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:27]:
```suggestion
* Updated the copy form dialog to use standard CMS patterns for button state and disable it after clicking [#1121](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1121).
```

File: 10/umbraco-forms/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,15 @@ This section contains the release notes for Umbraco Forms 8 and 10 including all
...
+* Ensured UI for the upload of a text file for a prevalue source only allows selection of expected .txt files.
+* Fixed handling of API and traditional form posts in reCAPTCHA 3 checks [#1150](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1150)
+* Fixed display of validation error when a duplicate form field alias is created [#1150](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1152)
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:30]:
```suggestion
* Fixed display of validation error when a duplicate form field alias is created [#1152](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1152)
```

File: 10/umbraco-forms/release-notes.md [hunk 3]
```diff
@@ -319,6 +328,13 @@ This section contains the release notes for Umbraco Forms 8 and 10 including all
 
 <summary>Version 8</summary>
 
+[**8.13.14**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.13.14) **(February 20th 2024)**
+
+* Null checks on setting prevalue captions handling issues with upgraded form definitions [#1148](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1148).
+* Fixed ordering of forms by name in security screen [#1122](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1122)
+* Update the copy form dialog to use standard CMS patterns for button state and disabling after click [#1121](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1121).
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:335]:
```suggestion
* Updated the copy form dialog to use standard CMS patterns for button state and disable it after clicking [#1121](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1121).
```

File: 12/umbraco-forms/release-notes.md [hunk 4]
```diff
@@ -17,6 +17,15 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 12 including all changes for this version.
 
+#### [**12.2.3**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F12.2.3) **(February 20th 2024)**
+
+* Ensured UI for the upload of a text file for a prevalue source only allows selection of expected .txt files.
```
Comment (eshanrnh) [inline 12/umbraco-forms/release-notes.md:22]:
```suggestion
* Ensured UI for the upload of a text file for a prevalue source only allows the selection of expected .txt files.
```

File: 10/umbraco-forms/release-notes.md [hunk 5]
```diff
@@ -319,6 +328,13 @@ This section contains the release notes for Umbraco Forms 8 and 10 including all
...
+* Ensured UI for the upload of a text file for a prevalue source only allows selection of expected .txt files.
```
Comment (eshanrnh) [inline 10/umbraco-forms/release-notes.md:336]:
```suggestion
* Ensured UI for the upload of a text file for a prevalue source only allows the selection of expected .txt files.
```

File: 12/umbraco-forms/release-notes.md [hunk 6]
```diff
@@ -17,6 +17,15 @@ If you are upgrading to a new major version, you can find information about the
...
+* Handled potential null value for prevalues for a form definition following an upgrade [#1157](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1157)
+* Fixed handling of API and traditional form posts in reCAPTCHA 3 checks [#1150](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1150)
+* Fixed display of validation error when a duplicate form field alias is created [#1150](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1152)
```
Comment (eshanrnh) [inline 12/umbraco-forms/release-notes.md:25]:
```suggestion
* Fixed display of validation error when a duplicate form field alias is created [#1152](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1152)
```

File: 13/umbraco-forms/release-notes.md [hunk 7]
```diff
@@ -17,6 +17,15 @@ If you are upgrading to a new major version, you can find information about the
 
 This section contains the release notes for Umbraco Forms 13 including all changes for this version.
 
+#### [**13.0.2**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.2) **(February 20th 2024)**
+
+* Ensured UI for the upload of a text file for a prevalue source only allows selection of expected .txt files.
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:22]:
```suggestion
* Ensured UI for the upload of a text file for a prevalue source only allows the selection of expected .txt files.
```

File: 13/umbraco-forms/release-notes.md [hunk 8]
```diff
@@ -17,6 +17,15 @@ If you are upgrading to a new major version, you can find information about the
...
+* Handled potential null value for prevalues for a form definition following an upgrade [#1157](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1157)
+* Fixed handling of API and traditional form posts in reCAPTCHA 3 checks [#1150](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1150)
+* Fixed display of validation error when a duplicate form field alias is created [#1150](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1152)
```
Comment (eshanrnh) [inline 13/umbraco-forms/release-notes.md:25]:
```suggestion
* Fixed display of validation error when a duplicate form field alias is created [#1152](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1152)
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
The PR looks great, @AndyButland 🙌 I've added a couple of suggestions.

**Comment by AndyButland** [comment]
All good with me, thanks - I've accepted the suggestions.

**Review by eshanrnh — APPROVED** [review]
Thanks for incorporating the suggestions. 🙌 Merging in the PR. 💪

**Pull request merged**